contracts/AINFT721Upgradeable.sol

```diff
@@ -0,0 +1,306 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+//import "@openzeppelin/contracts-upgradeable/token/ERC721/ERC721Upgradeable.sol";
```

Comment:
why commenting this out?

Reply:
why not using this? `https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/master/contracts/token/ERC721/extensions/ERC721BurnableUpgradeable.sol`

Reply:
I've already inherits ERC721EnumerableUpgradeable which contains ERC721Upgradeable, so I commented it out

Reply:
I change _burn() into burn() which is from `ERC721BurnableUpgradeable`.

Reply:
LGTM